docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -0,0 +1,136 @@
+
+### ESM configuration 
+The user identity mapping is defined as a distributed user identity filter, which is maintained by the System Authorization Facility (SAF) / External Security Manager (ESM).
+A distributed identity consists of two parts: a distributed identity name, and a trusted registry which governs that identity. 
+API ML provides a Zowe CLI plugin to help administrators to generate a JCL for creating the mapping filter specific for the ESM installed on the target mainframe system. 
+
+  See the [Identity Federation cli plugin]( )  <!--Add link --> documentation for details about how to use the plugin tool to set up the mapping in the ESM of your z/OS system.
+
+Alternatively, administrators can use the installed ESM functionality to create, delete, list, and query a distributed identity filter/s:
+ - For RACF consult [RACMAP command](https://www.ibm.com/docs/en/zos/2.3.0?topic=rcs-racmap-create-delete-list-query-distributed-identity-filter).
+ - For CA Top Secret use the [IDMAP Keyword - Implement z/OS Identity Propagation Mapping](https://techdocs.broadcom.com/us/en/ca-mainframe-software/security/ca-top-secret-for-z-os/16-0/administrating/issuing-commands-to-communicate-administrative-requirements/keywords/idmap-keyword-implement-z-os-identity-propagation-mapping.html).
+ - For CA ACF2 use [IDMAP User Profile Data Records](https://techdocs.broadcom.com/us/en/ca-mainframe-software/security/ca-acf2-for-z-os/16-0/administrating/administer-records/user-profile-records/idmap-user-profile-records.html).
+
+## API ML configuration
+Use the following procedure to enable the feature to use an OIDC Access Token as the method of authentication for the API Mediation Layer Gateway.
+
+ In the zowe.yaml file, configure the following properties:
+          
+   * **`components.gateway.apiml.security.oidc.enabled`**  
+   Specifies the global feature toggle. Set the value to true to enable OIDC authentication functionality.
+
+   * **`components.gateway.apiml.security.oidc.clientId`**  
+   Specifies the value of the client identification (client_id) assigned by the OIDC provider to the API ML Gateway.
+   
+   * **`components.gateway.apiml.security.oidc.clientSecret`**   
+   Specifies the client secret assigned by the OIDC provider to the API ML Gateway. It is used in combination with the client_id in Access Token validation requests at the /introspect endpoint of the OIDC provider.
+   
+   * **`components.gateway.apiml.security.oidc.registry`**  
+   Specifies the SAF registry used to group the identities recognized as having a OIDC identity mapping. The registry name is the string used during the creation of the mapping between the dustributed and mainframe user identities. See the [ESM configuration](#esm-configuration) for more information.
+
+   * **`components.gateway.apiml.security.oidc.introspectEndpoint`**
+
```

Comment:
Do we have a definition for this property?

Reply:
Added definition and links to the RFC

Reply:
I would mention that the default value is "/v1/introspect" ... that is what works for Okta.
Maybe also explain that this path portion is what comes after the issuer URI found in the token.